installers/rpm-code/codes/raydata.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+raydata_main() {
+    codes_dependencies common
+    install_pip_install \
+        ModestImage \
+        area_detector_handlers \
+        databroker-pack \
+        git+https://github.com/NSLS-II/eiger-io.git \
+        hdf5plugin \
+        pyOlog \
+        pychx \
+        xray-vision
+
+    raydata_patch
+}
+
+raydata_patch() {
+    local p=$(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+    for i in 'eiger_io fs_handler' 'modest_image modest_image' 'pyCHX chx_crosscor' 'xray_vision  __init__'; do
+        set -- $i
```

Comment:
Tricksy.

Reply:
A case of  "writing python in bash":). If you have a way you like more lmk.